compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/text/StringHelpers.skiko.kt

```diff
@@ -123,12 +123,12 @@ internal fun findNextNonWhitespaceSymbolsSubsequenceStartOffset(
 
     nextOffset = charIterator.next()
 
-    while (nextOffset != BreakIterator.DONE) {
+    while (nextOffset < currentText.length) { // charIterator.next() works one more time than needed, better use this
```

Comment:
Can we use `if (nextOffset != BreakIterator.DONE)` to make code a bit more elegant and get rid of the comment in current line?